src/cpp/library/string

```diff
@@ -1268,19 +1263,16 @@ char &string::operator[](size_t pos)
 
 char string::at(size_t pos) const
 {
-  __ESBMC_assert(
-    (pos >= 0) && (pos < this->_size), "Error! Invalid access memory area");
-  int i;
-  if (pos > this->length())
+  __ESBMC_assert((pos < this->_size), "Error! Invalid access memory area");
+
+  if (pos >= this->length())
```

Comment:
The assert and the if contradicts each other.

Reply:
Fixed.